app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :cart
+  helper_method :cart, :current_user
```

Comment:
We are going to need access to this in our views.

Reply:
This is a great catch!

Reply:
nice change